pom.xml

```diff
@@ -13,51 +13,39 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-	</properties>
+        <jdk.version>11</jdk.version>
+		<maven.compiler.source>${jdk.version}</maven.compiler.source>
```

Comment:
whitespace

Reply:
Pushed a change to substitute tabs with spaces